### DISCUSSION TITLE
Cannot submit ID proof for student pack verification

### DISCUSSION THREAD

**Discussion by luca009** [community/community#121838] [GitHub Education] [answered]
### Hi everyone,

Situation: I am trying to renew my student pack (third year now).
Going though the process on https://education.github.com/discount_requests/application, I can select my school (automatically found based on my associated school email address) and continue.

At this point, I am asked to upload proof of my academic status. I can upload an image of my student ID (which is still valid), but once I click "process my application", the page essentially just reloads. My uploaded image is gone, but no process request has gone through and no error message shows either. It just prompts me to upload the proof (again), no progress made.

Furthermore, I have tried different browsers and different devices to complete this process, including my phone, in case some sort of location check was failing. I've also tried using my phone's camera to capture the ID, though this failed, since the website's toggle between front-facing and back-facing camera does not work - it gets stuck on the front camera, which is useless for taking a photo of the ID, due to lack of autofocus.

**Comment by luca009** [comment, marked as answer]
I tried this again using my laptop camera and it worked this time, so I assume that this does only work when taking a photo directly. I haven't tested it, but the upload function might still be broken.

**Comment by sirpeep** [comment]
same thing happened to me recently, not sure if it went through or not lol. I'll try using my phone camera instead of a desktop

**Reply by sirpeep** [reply]
still didn't work for me, tried on windows 11 brave/firefox/chrome browsers and on iPhone 13 mini

**Comment by Pumpiron4u** [comment]
I'm experiencing an issue while trying to renew my student pack for the third year on the GitHub Education platform. After selecting my school and uploading proof of my academic status, clicking "process my application" just reloads the page without any progress or error messages. I've tried different browsers and devices, but the problem persists. Additionally, using my phone's camera to capture the ID has been unsuccessful due to a malfunction with the camera toggle. Any suggestions on how to resolve this issue would be greatly appreciated.
